kuma/payments/jinja2/payments/includes/payments-form.html

```diff
@@ -30,29 +32,36 @@
             {{form.stripe_token}}
             {{form.stripe_public_key}}
         {% endif %}
-
-
         <p class="legal-copy">
             Payments are not tax deductible. Your payment details will be processed by
             <a href="https://stripe.com/" rel="noopener" target="_blank">Stripe</a>.
             You can read our
             <a href="{{ url('payment_terms') }}" rel="noopener" target="_blank">Payment terms</a>,
             for more information.
         </p>
-
-        {% if recurring_payment %}
-            <input type="hidden" id="stripe_source_setup" value="1">
-            <label class="recurring-payment-confirmation">
-                {{form.accept_checkbox}}
-                <div class="control-indicator"></div>
+        <input type="hidden" id="stripe_source_setup" value="1">
+        {% set show_checkbox = recurring_payment or (is_popover and recurring_payment_enabled) %}
+        <div id="recurring-confirmation-container" {% if not show_checkbox %}class="hidden"{% endif %}>
+            <div class="recurring-payment-confirmation">
+                <label for="{{form.accept_checkbox.id_for_label}}">
+                    <span class="offscreen">
+                        By clicking this button, I authorize Mozilla to charge
+                        this payment method <span data-dynamic-amount>$64</span>
+                        each month, according to the <a href="{{ url('payment_terms') }}">Payment Terms</a>, until
+                        I cancel my subscription.
+                    </span>
```

Comment:
Is there a reason we are duplicating this copy? By marking this as offscreen, clicking on the "label" will also no longer cause the checkbox to be checked. It would be better for accessibility to keep this functionality.

If we cannot get this to work, we should probably mark this as `aria-hidden="true"`, which should prevent screen readers from reading the legal copy twice.

Reply:
We had a bug: clicking the link in the label would trigger the checkbox and not follow the href.
To solve this I moved the legal copy outside the label.
In order to ensure screenreaders read the copy with the context of the checkbox I've visually hidden the copy.

I think marking the copy outside of the label with `aria-hidden="true"` makes sense! I'll do that.